trace/opentelemetry/opentelemetry.go

```diff
@@ -235,26 +235,26 @@ func makeAttribute(key string, val interface{}) (attr attribute.KeyValue) {
 	switch v := val.(type) {
 	case string:
 		return attribute.String(key, v)
-	case []string:
-		return attribute.StringSlice(key, v)
+	// case []string:
```

Comment:
Do we need these commented cases? if no, let's remove them instead of commenting.

Reply:
We do need it... just that the version of otel we had to pull in doesn't support it.
Basically, I'll want to bring this back and just having the comments makes that simpler.